src/components/getData.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from 'react';
+
+function Quote() {
+  const [quote, setQuote] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+        headers: {
+          'X-Api-Key': 'WhmitRBUHv9LAjdRrvVfVw==QaSq4YojxGQkm1X6',
+        },
+      });
+      const fetchedQuote = await response.json();
+      setLoading(true);
+      setQuote(fetchedQuote[0].quote);
+    };
+
+    fetchQuote();
+  }, []);
+
+  if (loading === false) {
+    return (
+      <h1>loading</h1>
+    );
+  }
+  return (
+      <div>
+        {quote}
+      </div>
+  );
+}
+
```

Comment:
- [ ] Good job so far 👏🙌 However, I noticed that the code does not explicitly handle error states. In case the API request fails, it would be good to handle the error and provide appropriate feedback to the user. You can use a separate state variable to manage error states and display an error message accordingly ⛏🤓As an example, with the wrong API key, I'm getting the following error ⚠ Please make sure to implement it as mentioned in the project requirements

  ![image](https://github.com/sheezu900/Math-project/assets/106119105/e0a1cb37-7192-41b3-a4a3-7627fc7c022c)

_Project requirements 👇_
  
  ![image](https://github.com/sheezu900/Math-project/assets/106119105/5a4329b6-f3eb-4176-b426-f73e1985fe66)

- [OPTIONAL] I noticed that the code attempts to show a loading state when the loading state is false. Although it is working and your logic is correct, but the variable name is not according to the logic you are applying here. It's semantically more clear to use `if (loading)` to check if the loading state is `true` or you can use `if (isNotLoading === false)` for better readability.